deploy/templates/xml.tmpl

```diff
@@ -4,8 +4,8 @@
 <routes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xmlns="http://camel.apache.org/schema/spring"
         xsi:schemaLocation="
-            http://camel.apache.org/schema/spring
-            http://camel.apache.org/schema/spring/camel-spring.xsd">
+            https://camel.apache.org/schema/spring
```

Comment:
this need to be kept without the https, it is the namespace. This cannto be changed withotu breaking backward compatibility

Reply:
Fixed. Please, check again.